llvm/lib/Target/AVR/AVRTargetTransformInfo.h

```diff
@@ -0,0 +1,56 @@
+//===- AVRTargetTransformInfo.h - AVR specific TTI ---------*- C++ -*-===//
```

Comment:
Happened to find that this line is not aligned to 80 columns, though it is not a problem, it would be better to keep the same style with other files.